tests/php/tests/includes/test_class.wp-job-manager-usage-tracking-data.php

```diff
@@ -418,27 +331,69 @@ public function test_jobs_company_twitter() {
 	 * @covers WP_Job_Manager_Usage_Tracking_Data::get_usage_data
 	 */
 	public function test_jobs_company_video() {
-		$with_company_video_count = 3;
+		$published = 3;
+		$expired   = 2;
 
-		$this->factory->job_listing->create_many(
-			$with_company_video_count, array(
-				'meta_input' => array(
-					'_company_video' => 'youtube.com/1234',
-				),
-			)
-		);
+		$this->create_job_listings( '_company_video', 'youtube.com/1234', $published, $expired );
+
+		$data = WP_Job_Manager_Usage_Tracking_Data::get_usage_data();
+		$this->assertEquals( $published + $expired, $data['jobs_company_video'] );
+	}
 
-		// Add 5 with no company video
-		$this->factory->job_listing->create( array() );
-		foreach ( array( '', '   ', "\n\t", " \n \t " ) as $val ) {
+
+	/**
+	 * Creates job listings with the given meta values. This will also create
+	 * some listings with values for the meta parameter that should be
+	 * considered empty (e.g. spaces) and some entries with other statuses
+	 * (such as draft). For tracking data, only the published and expired
+	 * entries should be counted.
+	 *
+	 * @param string $meta_name  the name of the meta parameter to set.
+	 * @param string $meta_value the desired value of the meta parameter.
+	 * @param int $published     the number of published listings to create.
+	 * @param int $expired       the number of expired listings to create.
+	 */
+	private function create_job_listings( $meta_name, $meta_value, $published, $expired ) {
+		// Create published listings.
+		$this->factory->job_listing->create_many( $published, array(
+			'meta_input' => array(
+				$meta_name => $meta_value,
+			),
+		) );
+
+		// Create expired listings.
+		$this->factory->job_listing->create_many( $expired, array(
+			'meta_input' => array(
+				$meta_name     => $meta_value,
+				'_job_expires' => "2018-01-01",
```

Comment:
I'm not sure this will actually set the status to expired. The post status may need to be explicitly set to Expired?

Reply:
That's right. This is changed in a later commit 🙂